components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/internal/IdentityEventServiceComponent.java

```diff
@@ -29,7 +29,7 @@
 import org.wso2.carbon.identity.event.handler.AbstractEventHandler;
 import org.wso2.carbon.identity.event.services.IdentityEventService;
 import org.wso2.carbon.identity.event.services.IdentityEventServiceImpl;
-import org.wso2.carbon.idp.mgt.IdpManager;
+//import org.wso2.carbon.idp.mgt.IdpManager;
```

Comment:
remove commented lines if not required

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/3518/commits/65c22a6cceddbd6bde83d814e81c53535405398d